pkgs/tools/text/mark/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, stdenv, buildGoModule, fetchFromGitHub }:
```

```suggestion
{ lib, buildGoModule, fetchFromGitHub }:
```

I don't believe it's used.